src/SQLChecks/Tests/SQLInstance.tests.ps1

```diff
@@ -49,4 +49,10 @@ Describe "Database Mail is configured" -Tag DatabaseMail {
 	It "Database mail is running on $serverInstance" {
 		@(Get-DatabaseMailStatus -Config $Config).Status | Should Be "STARTED"
 	}
+}
+
+Describe "SQL Alerts" -Tag SQLAlerts {
```

Comment:
Could we make this more descriptive - as it's unique on the module and frequently in config files with little context the more verbose the better.  UnconfiguredSQLAgentAlerts maybe?